client-app/ui-kit/components/atoms/checkbox/vc-checkbox.vue

```diff
@@ -23,6 +23,7 @@
         class="vc-checkbox__input"
         :data-test-id="testId"
         @change="change"
+        @click="clickHandler"
```

Comment:
onClick

Reply:
onclick is a native js handler, in Vue it's @click ?

Reply:
I mean function name. If several things inside it is good to name it @click="onClick"